### DISCUSSION TITLE
What's the most performant folder sharing method (for my case)

### DISCUSSION THREAD

**Discussion by unlocomqx** [lima-vm/lima#1151] [Q&A] [answered]
Hi,

I was running an apache dev server locally (MacOS Ventura 13.0) and it was running slow
(I use PrestaShop mostly, it's read/write heavy)

I then switched to Lima and shared my dev folder as you would, but I noticed no real gains.

I made a test without folder sharing (I simply installed PrestaShop on /var/www/html inside the instance) and the difference was **huge**
I then mounted /var/www/html using sshfs using something like
```shell
sshfs -p 60022 root@127.0.0.1:/var/www/html ./ubuntu -o allow_other,defer_permissions
```

It's fine, but I thought maybe there's a more performant way to share a folder without losing server performance

Any hint is appreciated

Thank you very much

**Comment by afbjorklund** [comment]
You can take a look at [mutagen.io](https://mutagen.io/), or other similar sync solution ?

**Comment by unlocomqx** [comment]
Thank you, I'll look into it and check if it fits my needs

**Comment by AkihiroSuda** [comment, marked as answer]
You may also want to try `mountType: virtiofs` with `vmType: vz`:
- https://github.com/lima-vm/lima/pull/1147

Coming in Lima v0.14

**Comment by unlocomqx** [comment]
@AkihiroSuda Thank you, I will check it out
